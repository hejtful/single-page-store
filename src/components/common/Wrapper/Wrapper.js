import React from 'react';

import styles from './Wrapper.module.css';

export const Wrapper = ({ children }) => (
  <main className={styles.wrapper}>{children}</main>
);
