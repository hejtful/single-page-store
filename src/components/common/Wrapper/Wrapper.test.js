import React from 'react';
import { render } from '@testing-library/react';
import { Wrapper } from './Wrapper';

it('renders wrapper element and all children', () => {
  const { container } = render(
    <Wrapper>
      <div>Test child 1</div>
      <div>Test child 2</div>
    </Wrapper>
  );

  expect(container).toMatchSnapshot();
});
