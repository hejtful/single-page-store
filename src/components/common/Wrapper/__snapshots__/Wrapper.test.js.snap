// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders wrapper element and all children 1`] = `
<div>
  <main
    class="wrapper"
  >
    <div>
      Test child 1
    </div>
    <div>
      Test child 2
    </div>
  </main>
</div>
`;
